_shows/14_15/bent.md

```diff
@@ -77,6 +77,17 @@ assets:
     video: HcSb3w6
     title: Backstage At
 
+links:
+  - type: Review
+    href: http://www.impactnottingham.com/2014/11/bent-nottingham-new-theatre/
+    snapshot: SSTKV
+    publisher: Impact Nottingham 
+    author: Olivia Rook
+    title: "Bent @ Nottingham New Theatre"
+    date: 2014-11-20
+    rating: 9
```

Comment:
This is the cause of the error, rating needs to be in format `x/of_y`. In this case should be `9/10`.